visualization/src/point_picking_event.cpp

```diff
@@ -109,22 +109,10 @@ pcl::visualization::PointPickingCallback::Execute (vtkObject *caller, unsigned l
     }
     else if (eventid == vtkCommand::LeftButtonReleaseEvent)
     {
-      vtkActorCollection *selectedActorsCollection = vtkActorCollection::New();
       style->OnLeftButtonUp ();
-      std::vector<std::vector<int> > indices;
-      int nb_points = performAreaPick (iren, indices, selectedActorsCollection);
-      pcl::visualization::CloudActorMapPtr cam_ptr = style->getCloudActorMap();
-      
       std::map<std::string, std::vector<int>> cloudIndices;
-      selectedActorsCollection->InitTraversal();
-      for(vtkIdType i = 0; i< selectedActorsCollection->GetNumberOfItems(); i++)
-      {
-        //vtkActor* actor = actors_->GetNextActor();
-        vtkSmartPointer<vtkActor> actor_ptr = selectedActorsCollection->GetNextActor();
-        const auto actor = std::find_if(cam_ptr->cbegin(), cam_ptr->cend(), [&actor_ptr](const auto& cloud_actor) { return cloud_actor.second.actor == actor_ptr; });
-        const std::string name = (actor != cam_ptr->cend()) ? actor->first : "not_found";
-          cloudIndices.insert({name, std::move(indices[i])});
-      }
+      pcl::visualization::CloudActorMapPtr cam_ptr = style->getCloudActorMap();
+      int nb_points = performAreaPick (iren, cam_ptr, selectedActorsCollection);
```

Comment:
Did you mean to say this?
```suggestion
      int nb_points = performAreaPick (iren, cam_ptr, cloudIndices);
```